core/frontend/src/utils/mavlink_prettifier.ts

```diff
@@ -0,0 +1,45 @@
+import { Dictionary } from '@/types/common'
+
+const formatters = {
+  STATUSTEXT(message: any): string {
```

Comment:
I would just recommend to move it in alphabetic order before merging, this will probably grow a lot soon.

Reply:
good point, will do